README.md

```diff
@@ -41,11 +41,6 @@ A live demonstration of my bot can be viewed [here.](https://twitter.com/tinyflo
 
 Should you host your own variation of this bot, please include the link to my account in the biography of the account. The `config` dictionary can be freely adjusted to your needs.
 
-## Contributors
-
-* **Panagiotis Vasilopoulos** - *Initial work, maintainer.* - [AlwaysLivid](https://alwayslivid.com)
-* **karl01101101** - *Worked on day randomization* - [GitHub](https://github.com/karl01101101)
-
```

Comment:
Please keep all README-related changes separate (as in in a different pull request, since updating the README is not relevant to resolving #19). You're also proposing to change the structure of the configuration file in a way that I'd accept-- please include that change in your changes.